### PR TITLE
Swedish lang 

### PR DESCRIPTION
**Pull request by Miya25**
Translated By RanveerSoni

### CODE REVIEW

File: i1n8.config.js [hunk 1]
```diff
@@ -5,10 +5,11 @@ module.exports = {
     'Spanish',
     'German',
     'Italian'
```
Comment (TheRealToxicDev) [inline i1n8.config.js:7]: Please correct the missing comma here.

Comment (Miya25) [reply i1n8.config.js]: oh sorry fixed

**Review by SerenModz21 — CHANGES_REQUESTED** [review]

**Comment by Miya25** [comment]
sorry i am done all the things thanks for letting me know!

File: i18n/locales/Swedish/Swedish.tsx [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+                    description: "Vi samlar in och lagrar flera olika typer av information som ditt Google-användarnamn, e-post, Avatar, 2Factor Status och Authorization Token.",
+                },
+                data_usage: {
+                    title: "Hur vi använder din information",
+                    description: "Vi använder informationen vi samlar in på olika sätt. Inklusive till:",
+                    list_items: {
+                        item_1: "Tillhandahålla, driva och underhålla vår webbplats och tjänster.",
+                        item_2: "Förbättra, anpassa och utöka vår webbplats och tjänster.",
+                        item_3: "Förstå och analysera hur du använder och interagerar med vår webbplats.",
+                        item_4: "Utveckla och implementera nya produkter, tjänster, funktioner och funktionalitet."",
```
Comment (SerenModz21) [inline i18n/locales/Swedish/Swedish.tsx:276]:
```suggestion
                        item_4: "Utveckla och implementera nya produkter, tjänster, funktioner och funktionalitet.",
```

**Review by SerenModz21 — CHANGES_REQUESTED** [review]

File: i18n/locales/Swedish/Swedish.tsx (hunk 2 above)
Comment (SerenModz21) [reply i18n/locales/Swedish/Swedish.tsx:276]: once this one is resolved, please request a new review from me

**Review by RizonFTW — APPROVED** [review]
Seems fine to me now. Thank you.

**Review by SerenModz21 — APPROVED** [review]
looks good

**Pull request merged**